lib/charms/operator_libs_linux/v0/apt.py

```diff
@@ -126,6 +129,8 @@
 # to 0 if you are raising the major API version
 LIBPATCH = 17
```

Comment:
This should be bumped - I think the practice is that every improvement gets a release published, because there are so few here.